docker-bake.hcl

```diff
@@ -0,0 +1,76 @@
+variable "GO_VERSION" {
+  default = "1.16"
+}
+variable "BIN_OUT" {
+  default = "./bin"
+}
+
+group "default" {
+  targets = ["binaries"]
+}
+
+group "validate" {
+  targets = ["lint", "vendor-validate", "docs-validate"]
+}
+
+target "lint" {
+  args = {
+    GO_VERSION = GO_VERSION
+  }
+  dockerfile = "./hack/dockerfiles/lint.Dockerfile"
+  output = ["type=cacheonly"]
+}
+
+target "vendor-validate" {
+  args = {
+    GO_VERSION = GO_VERSION
+  }
+  dockerfile = "./hack/dockerfiles/vendor.Dockerfile"
+  target = "validate"
+  output = ["type=cacheonly"]
+}
+
+target "vendor-update" {
+  args = {
+    GO_VERSION = GO_VERSION
+  }
+  dockerfile = "./hack/dockerfiles/vendor.Dockerfile"
+  target = "update"
+  output = ["."]
+}
+
+target "test" {
+  args = {
+    GO_VERSION = GO_VERSION
+  }
+  target = "test-coverage"
+  output = ["./coverage"]
+  platforms = [
+//    "darwin/amd64",
+//    "darwin/arm64",
+    "linux/amd64",
```

Comment:
Containerized tests are only available for linux platform (that's why a specific job has been added in the build workflow).